templates/header-user.cfg.xml.erb

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- http://wiki.openrepose.org/display/REPOSE/Header+Identity -->
+<header-user xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+                 xmlns='http://docs.openrepose.org/repose/header-user/v1.0'
+                 xsi:schemaLocation='http://docs.openrepose.org/repose/header-user/v1.0 ../config/header-identity-configuration.xsd'>
+
+<%- if @source_headers -%>
```

Comment:
@adrianjgeorge what happens to repose if source_headers is an empty list and so this block is left out but the file exists with the `header-user` structure ?

Reply:
It'll reject it. It's a required field. Should probably add a check in the class to make sure it's not empty. It's a weird pointless extra element, I'll make a note to deprecated and remove it in future versions.

Reply:
So based on that the filter should validate that the `source_headers` is not empty and error if it is.  The default value should probably change from `[ ]` to `undef` too

Reply:
:+1: will do

Reply:
Since source headers is required now, this check is no longer neccessary

Reply:
+1 that conditional can go away